Automattic/jetpack
# My Jetpack: reorganizing icons - tweak icons for interstitial pages

This PR reorganizes the icons in order to use and tweak their size easily. It's needed to adjust the size of the icons for the interstitial pages.

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: re organize icons - tweak icons for interstitial pages

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to My Jetpack.
* Confirm visually that icons are properly rendered in every card.
<img width="1157" alt="image" src="https://user-images.githubusercontent.com/77539/153489329-91794c45-da04-459f-89bf-d220f01d669c.png">
* Confirm the icon for the interstitial page, for instance `https://your-testing-site/wp-admin/admin.php?page=my-jetpack#/add-search`

<img width="674" alt="Screen Shot 2022-02-10 at 5 17 02 PM" src="https://user-images.githubusercontent.com/77539/153489498-a2c56f07-13b6-40b2-9eec-4c20f1d483ad.png">

**Storybook**

* Run storybook

```bash
cd ./jetpack/projects/js-packages/storybook
```

```bash
> pnpm install & pnpm storybook:dev
```

Take a look at the Icons section

<img width="1118" alt="image" src="https://user-images.githubusercontent.com/77539/153492069-e09991bb-2de4-4449-a4d2-916710aaa914.png">
